inferlets/mcts-python/main.py

```
@@ -0,0 +1,332 @@
+    ctx.user(f"Problem:\n{problem}\n\nBest reasoning path:\n{path_block(best_path)}\n\nFinal answer:")
+    ctx.cue()
+    text = await ctx.generate(Sampler.argmax(), max_tokens=final_tokens).collect_text()
+    return text.strip()
+
+
+# =============================================================================
+# Entry point
+# =============================================================================
+
+
+async def main(input: dict) -> str:
```

Ruff (0.15.15)

[error] Function argument `input` is shadowing a Python builtin

(A002)